### DISCUSSION TITLE
Allow to delete multiple objects while holding delete key in editor

### DISCUSSION THREAD

**Discussion by ChenMerami** [ppy/osu#29514] [Ideas]
Allow to delete multiple objects while holding delete key in editor (middle mouse button). Here is an example of this behavior in other software

https://github.com/user-attachments/assets/45e84073-146c-42c1-aa3b-2ce83d0d340b

**Comment by vatei** [comment]
Sounds like a good idea! 
As an alternative I thought about middle mouse drag to select+delete objects but yours might be nicer

**Comment by peppy** [comment]
I'm fine with this being added for middle mouse button specifically (not for the key bind though, feels weird). Generally known as "painting" UX, useful for deleting / tagging / selecting.
